### DISCUSSION TITLE
Why is the "Last seen" status of all my devices "N/A"?

### DISCUSSION THREAD

**Discussion by bikeymouse** [Koenkk/zigbee2mqtt#18958] [General]
Hi,
I'd like to see when a device last had contact with the coordinator, which I thought would be shown in the front-end in the "About" tab  "Last seen" of the device. However on all my devices it is showing "N/A" .
Something wrong in my config?
<img width="917" alt="Zigbee2mqtt" src="https://github.com/Koenkk/zigbee2mqtt/assets/34967010/f9c4ba37-d11b-47bf-89cc-55dd154279b2">

**Comment by keithr59** [comment]
You need to enable **Last seen** in Advanced Settings.

![image](https://github.com/Koenkk/zigbee2mqtt/assets/93706044/c64c4cba-e295-4dc5-a234-1cbfc99178ed)

**Comment by bikeymouse** [comment]
Ah, yes found it thanks!
Is there any reason why this is not on by default? Does this pose additional burden on the network?

**Reply by keithr59** [reply]
I've had it enabled for nearly two years and it doesn't appear to have any impact but then I only have 20 or so devices

**Reply by DubhAd** [reply]
As far as I can tell no extra traffic is caused by this, it just reports what already exists.
